src/widgets/RainWidget.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import GenericWidget from './GenericWidget';
+import './RainWidget.css';
+
+class RainWidget extends Component {
+
+  render() {
+    return (
+      <GenericWidget class="RainWidget" icon="cloud"
+          isOn={this.props.rain > 0}
+          value1={<div className="rain">{this.props.rain}</div>}
+          value2={<div className="rate">{this.props.rate}</div>}
```

Comment:
Yeah this was poor design, I should add a value1 class to the h2. Thanks for the update.